source/content/guides/edge-integrations/04-plugin-install.md

```diff
@@ -71,11 +71,18 @@ If the plugin is installed and configured correctly, the `Audience` or `Audience
 
 ### Geolocation Code Samples
 
+Edge Integrations can easily integrate with various WordPress plugins.
 
+[Advanced Custom Fields (ACF)](https://www.advancedcustomfields.com/) is a popular WordPress plugin that allows developers to turn sites into a fully-fledged content management system. [CMB2](https://github.com/CMB2/CMB2) is a developer's toolkit for building metaboxes, custom fields, and forms for WordPress. 
+
+We've created example plugins to integrate with each of these third-party plugins, where the given solution is used to render content based on a visitor's location.
+
+- [Geo ACF example plugin](https://github.com/pantheon-systems/wp13n/tree/master/wp-content/plugins/geo-acf)
+- [Geo CMB2 example plugin](https://github.com/pantheon-systems/wp13n/tree/master/wp-content/plugins/geo-cmb2)
```

Comment:
the WP13N repo is private, so these can/should be broken into their own public repositories.

You can ping me if you want to try to retain the git history of these folders with `git filter-repo`

Reply:
Created new repo and linked in [031590f](https://github.com/pantheon-systems/documentation/pull/7199/commits/031590faae04c2637851dd1da5cf7af54a881f95).